README.md

```diff
@@ -12,6 +12,11 @@ Read more about the [IAMC data format](https://docs.ece.iiasa.ac.at/iamc.html).
 This project uses the Python package [nomenclature-iamc](https://nomenclature-iamc.readthedocs.io)
 for management of codelists and validation of scenario data in the IAMC data format.
 
+> [!TIP]
```

Comment:
Thanks for the addition, but I'd put that below the more user-friendly download-excel-file-driectly tip.